### PR TITLE
Update to use Kind v0.4.0 and remove a todo

### PR DESCRIPTION
**Pull request by shashidharatd**
**What this PR does / why we need it**:
Handle a todo item.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
https://github.com/kubernetes-sigs/kind/issues/111

**Special notes for your reviewer**:
/cc @font 
**Release note**:

```release-note

```

### CODE REVIEW

File: scripts/create-clusters.sh [hunk 1]
```diff
@@ -139,21 +139,14 @@ function fixup-cluster() {
   local i=${1} # cluster num
 
   local kubeconfig_path="$(kind get kubeconfig-path --name cluster${i})"
+  # Set container IP address as kube API endpoint in order for clusters to reach kube API servers in other clusters.
+  kind get kubeconfig --name "cluster${i}" --internal >${kubeconfig_path}
```
Comment (font) [inline scripts/create-clusters.sh]: Will this work for MacOS? The `--internal` flag according to the PR will use the internal container IP address. I believe MacOS relies on the use of `localhost`, which is what kind uses by default i.e. without `--internal` flag. However, I cannot verify this.

Comment (shashidharatd) [reply scripts/create-clusters.sh]: @font, Ideally this should be taken care by kind, to implement a feature consistently across platform. Also theoretically the same problem would be present if `localhost` is used in `MacOS`. I too am not familiar with `MacOS`. Can somebody using `MacOS` confirm this please. @xunpan @aoxn @iamzhout can you please help in verifying this issue.

Comment (shashidharatd) [reply scripts/create-clusters.sh]: Thanks @font for the clarification. I have updated the code. PTAL once again. Thanks!

**Comment by marun** [comment]
@xunpan kind on macos is a [documented special-case](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/cluster-registration.md#joining-kind-clusters-on-macos)

**Pull request merged**